agent/command/s3_copy.go

```diff
@@ -346,8 +346,9 @@ func (c *s3copy) attachFiles(ctx context.Context, comm client.Communicator,
 	}
 	logger.Execution().Infof("Attaching file '%s'.", displayName)
 	file := artifact.File{
-		Name: displayName,
-		Link: fileLink,
+		Name:        displayName,
```

Comment:
GH doesn't let me comment on unchanged lines, but is [the S3 bucket](https://github.com/evergreen-ci/evergreen/blob/d7daa79cf9adb8145cc0bbe23905bb320d05c150/agent/command/s3_copy.go#L274-L279) also supposed to set the content type for uploading files? Seems like if we were to use the content type for our own purposes, we might as well set it for the actual S3 file as well.

Reply:
I removed the changes for s3 copy.